Tests/MongoSwiftSyncTests/UnifiedTestRunner/UnifiedTestRunner.swift

```diff
@@ -126,8 +126,9 @@ struct UnifiedTestRunner {
                     }
                 }
 
-                for operation in test.operations {
-                    try operation.executeAndCheckResult(entities: &entityMap)
+                for (i, operation) in test.operations.enumerated() {
+                    let context = Context(["Operation \(i)"])
```

Comment:
I think it might be useful to include the operation name here in addition to its index.

Reply:
done. I ended up just storing the operation name on the `UnifiedOperation` object now, since I thought that would be a little clearer and easier to match up with the yaml/JSON than using the name of the Swift type.